pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -1075,6 +1087,8 @@ func TestReconcile(t *testing.T) {
 						"/tekton/tools/0",
 						"-termination_path",
 						"/tekton/termination",
+						"-timeout",
+						"",
```

Comment:
what do you think about having this be the default and not provide it when it's not needed? (no strong feelings but feels a bit unnecessary)